tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "compilerOptions": {
-    "jsx": "preserve",
+    "jsx": "react-jsx",
+    "jsxImportSource": "preact",
+    "types": ["preact"],
```

Comment:
This brings the repo up to date with latest jsx-runtimes.

This is compatible with the latest jsx-runtime stuff. 

https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

https://github.com/preactjs/preact/releases/tag/10.5.0